integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -1718,6 +1733,13 @@ def do_it(spark):
     assert_gpu_and_cpu_are_equal_collect(do_it,
         conf={'spark.sql.ansi.enabled': 'true'})
 
+@ignore_order(local=True)
+@pytest.mark.parametrize('data_gen', [ArrayGen(sub_gen, nullable=False) for sub_gen in all_basic_gens + decimal_gens])
+def test_grpby_array_agg_on_func(data_gen):
+    gen = RepeatSeqGen(data_gen, length=10)
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: two_col_df(spark, gen, IntegerGen())
+            .groupBy('a').agg(f.first('b'), f.last('b'), f.max('b'), f.min('b')))
```

Comment:
The test above is almost identical to this one. It has some special cases for first and last because.

"coalescing because first/last are not deterministic" we probably need to do the same thing here.

Reply:
I have removed this test as it was testing something that didn't need to be tested